espnet2/asr/espnet_model.py

```diff
@@ -46,8 +49,9 @@ def __init__(
         encoder: AbsEncoder,
         decoder: AbsDecoder,
         ctc: CTC,
-        rnnt_decoder: None,
+        transducer_decoder: Optional[Tuple[AbsDecoder, torch.nn.Module]],
```

Comment:
Unreadable. Split it into two arguments: `transducer_decoder`, `joint_net`

Reply:
OK!

Reply:
Out of curiosity, why do you think it's "unreadable"? I'm not too experienced with type hints yet so I'm trying to understand!

Reply:
This argument represents two independent objects. There are no reason to pack it into one and it's unexpected, normally.